src/applications/vaos/new-appointment/components/CommunityCareProviderSelectionPage/ProviderSelectionField.jsx

```diff
@@ -125,83 +147,123 @@ function ProviderSelectionField({
           <h2 className="vads-u-font-size--h3 vads-u-margin-top--0">
             Choose a provider
           </h2>
-          <p>Your address on file:</p>
-          <ResidentialAddress address={address} />
+          {!loadingProviders && (
```

Comment:
Seems like we're still showing the residential address, even when the sorting is by current location. This screen shows the address hidden: https://adhoc.invisionapp.com/share/BFZED2H362S#/screens/438121104

Reply:
Done.